webdev/static/function.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
+self.__PROJECT_TYPE_PATTERN = "{{ .pathFull }}";
```

Comment:
1 - backblaze full path
2 - the idea is to use the template in golang, doing replace

Reply:
@augustollenz this is following the golang template layout. It can be easily replaced by the cli.

Reply:
@maxwelbm About the item 1, I don't understand what you mean by "backblaze full path". Anyway, I think that we should use `self.__PROJECT_TYPE_PATTERN = PROJECT_TYPE:STATIC_SITE`, maybe this can even be hardcoded in JS code.

See https://github.com/aziontech/azion-framework-adapter/blob/main/src/models/builders/static-site-builder.ts#L30

The only use of this variable is to enable us to count how many edge-functions were deployed using this template. So it's important to keep using the same identifier so that the pattern matching used to count this doesn't need to be updated.

Reply:
> @augustollenz this is following the golang template layout. It can be easily replaced by the cli.

Ok @PatrickMenoti.

From my point of view, it seems more natural to keep this stuff here instead of CLI and just use JS native tooling to inject `version-id` variable. This would keep the coupling between azion-cli and azioncli-template minimal.

That said, I'm ok with the proposed approach.